nsdb-sql/src/main/scala/io/radicalbit/nsdb/sql/parser/SQLStatementParser.scala

```diff
@@ -126,11 +126,13 @@ final class SQLStatementParser extends RegexParsers with PackratParsers with Reg
 
   // def instead val because timestamp should be calculated everytime but there is a problem with
   // more than one "now - something" in the same query because "now" will be different for each condition
-  private def delta: PackratParser[RelativeComparisonValue[Long]] = now ~> ("+" | "-") ~ longValue ~ timeMeasure ^^ {
-    case "+" ~ v ~ ((unitMeasure, timeInterval)) =>
+  private def delta: PackratParser[ComparisonValue[Long]] = now ~> ("+" | "-").? ~ longValue.? ~ timeMeasure.? ^^ {
```

Comment:
maybe it' s possible to use only one option. Something like 

`now ~> (("+" | "-") ~ longValue).?`

Reply:
in the current implementation a query like this would be allowed

`where timestamp > now + `

Reply:
Yes! New comming is coming...